### PR TITLE
Docusaurus

### PR DESCRIPTION
**Pull request by samikuikka**
Documentation site with docusaurus added to the project.

### CODE REVIEW

**Comment by jniemii** [comment]
Nitpick: I think the main README should have a hint for how to find the documentation (just go to docusaurus/?).

**Review by KorpelainenTommi — APPROVED** [review]
Looks fine

File: docusaurus/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+### Build
+
+```
+$ yarn build
+```
+
+This command generates static content into the `build` directory and can be served using any static contents hosting service.
+
+### Deployment
```
Comment (jaakkoha-bytecraft) [inline docusaurus/README.md:25]: These aren't correct for our case, remove this section.

**Review by itulkki — COMMENTED** [review]
Should be fine after fixing docusaurus/README.md according to Jaakko's comments.

File: docusaurus/README.md (hunk 1 above)
Comment (KorpelainenTommi) [reply docusaurus/README.md:25]: Replaced sections about yarn with documentation taken from https://docusaurus.io/docs

**Pull request merged**